x-pack/plugins/security_solution/public/management/links.ts

```diff
@@ -180,6 +186,23 @@ export const links: LinkItem = {
       experimentalKey: 'riskScoringRoutesEnabled',
       licenseType: 'platinum',
     },
+    {
+      id: SecurityPageName.entityAnalyticsAssetClassification,
+      title: ASSET_CRITICALITY,
+      description: i18n.translate(
+        'xpack.securitySolution.appLinks.assetClassificationDescription',
+        {
+          defaultMessage: 'Represents the criticality of an asset to your business infrastructure.',
+        }
+      ),
+      landingIcon: IconAssetCriticality,
+      path: ENTITY_ANALYTICS_ASSET_CRITICALITY_PATH,
+      skipUrlState: true,
+      hideTimeline: true,
+      capabilities: [`${SERVER_APP_ID}.entity-analytics`],
+      // experimentalKey: 'riskScoringRoutesEnabled',
```

Comment:
should this be commented out?

Reply:
Yes! I am working on this change on a different branch. Thanks for noticing it.